lnd_test.go

```diff
@@ -574,88 +655,223 @@ func testMultiHopPayments(net *networkHarness, t *testing.T) {
 	assertAsymmetricBalance(net.Bob, &aliceFundPoint, sinkBal, sourceBal)
 
 	ctxt, _ = context.WithTimeout(ctxb, timeout)
-	closeChannelAndAssert(t, net, ctxt, net.Alice, chanPointAlice)
+	closeChannelAndAssert(ct, net, ctxt, net.Alice, chanPointAlice)
 	ctxt, _ = context.WithTimeout(ctxb, timeout)
-	closeChannelAndAssert(t, net, ctxt, carol, chanPointCarol)
+	closeChannelAndAssert(ct, net, ctxt, carol, chanPointCarol)
+}
+
+// testMaxPendingChannels checks that in case excess max pending channels for
+// one peer error is returned to the RPC client (in case of
+// local open channel initiation) and to the remote peer (in case of remote
```

Comment:
As mentioned above, the scope of this test can be reduced to only test against the case a remote (inbound) max pending channel violation.